### PR TITLE
[WIP]This PR will fix Keypairs failure.

### PR DESCRIPTION
**Pull request by sbulage**
{{ pytest: cfme/tests/cloud/test_keypairs.py -v --use-provider rhos11 --long-running }}

### CODE REVIEW

File: cfme/cloud/keypairs.py [hunk 1]
```diff
@@ -128,8 +128,12 @@ class KeyPair(BaseEntity, WidgetasticTaggable):
...
     def delete(self, cancel=False, wait=False):
         view = navigate_to(self, 'Details')
-        view.toolbar.configuration.item_select('Remove this Key Pair',
-                                               handle_alert=(not cancel))
+        if self.appliance.version < "5.8":
+            view.toolbar.configuration.item_select('Remove this Key Pair',
+                                                   handle_alert=(not cancel))
+        else:
+            view.toolbar.configuration.item_select('Remove this Key Pair from Inventory',
+                                                   handle_alert=(not cancel))
```
Comment (mfalesni) [inline cfme/cloud/keypairs.py]: **OPTIONAL**: You can get rid of the parentheses around `not cancel`

Comment (sbulage) [reply cfme/cloud/keypairs.py]: Done.

File: cfme/cloud/keypairs.py [hunk 2]
```diff
@@ -128,8 +128,12 @@ class KeyPair(BaseEntity, WidgetasticTaggable):
 
     def delete(self, cancel=False, wait=False):
         view = navigate_to(self, 'Details')
-        view.toolbar.configuration.item_select('Remove this Key Pair',
-                                               handle_alert=(not cancel))
+        if self.appliance.version < "5.9":
```
Comment (mshriver) [inline cfme/cloud/keypairs.py]: **Required**
I would rather see a single call to item_select, and construction of the message based on version before the call.

example of how I made a very similar change:
https://github.com/ManageIQ/integration_tests/pull/5815/files

Comment (sbulage) [reply cfme/cloud/keypairs.py]: Ok will update the code.

**Comment by sbulage** [comment]
#5850

**Comment by mshriver** [comment]
@sbulage just waiting on PRT now? #5850 meged

**Comment by sbulage** [comment]
Yes !!

**Comment by sbulage** [comment]
@mshriver Locally all tests are passing, don't know why failing on PRT with same result..?

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 0edc42f7cee3c560e9c3af7a418d6d72c020c7c2:

`cfme/cloud/keypairs.py`:
- :red_circle: Line 244:12: **F821** *undefined name 'TagPageView'*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by sbulage** [comment]
This test case has been fixed. Closing this PR.

**Pull request closed without merging**